src/commands/help.ts

```diff
@@ -55,10 +55,58 @@ export class HelpCommand extends Command {
    */
   private async showHelp(msg: Message, embedArg: MessageEmbed, showAll: boolean): Promise<void> {
     let embed = embedArg.setTitle('Command List').setDescription(this.getDescription(msg))
-
     let length = embed.title.length + (embed.footer.text || '').length + embed.description.length
 
-    for (const command of this.client.commands.values()) {
+    const commands = this.getCommandsByGroup()
+
+    for (const pair of commands.entries()) {
```

Comment:
I believe this can be destructured using `for (const [key, value] of commands) {`

Test that for me, and let me know if that works.

Reply:
It does, I'm opening a PR now containing this adjustment and a fix(?) for the magic string problem.